test/unit/ui/camera.test.js

```diff
@@ -1136,5 +1136,38 @@ test('camera', (t) => {
         t.end();
     });
 
+    t.test('#fitBounds', (t) => {
+        t.test('no padding passed', (t) => {
+            const camera = createCamera();
+            const bb = [[-133, 16], [-68, 50]];
+
+            camera.fitBounds(bb, {duration:0});
+            t.deepEqual(fixedLngLat(camera.getCenter()), { lng: -100.5, lat: 34.717077774 }, 'pans to coordinates based on fitBounds');
```

Comment:
`fixedLngLat` does round to a default precision of 9 decimal places – that is rather precise, though, so it might be a good idea to reduce precision a bit